valgrind.xml

```diff
@@ -0,0 +1,3565 @@
+<?xml version="1.0"?>
```

Comment:
I don't think this is supposed to be checked in, maybe add to .gitignore?

Reply:
Oof, I don't know how that got in. Will remove.